From defc37e5972df5a5a476d422efd39a914dafbc6f Mon Sep 17 00:00:00 2001
From: Jin Xue <jin.xue@intel.com>
Date: Wed, 4 Sep 2019 10:52:25 +0800
Subject: [PATCH 6/6] eliminate building errors

eliminate compiling errors such as "unused variable"

Change-Id: I7aca7e739a8c91cec697be3e628e368832a90d6f
Tracked-On: ACP-740
Signed-off-by: Jin Xue <jin.xue@intel.com>
Reviewed-on: 679515
Reviewed-by: Yang, Bin Y <bin.y.yang@intel.com>
---
 audio_hw.c | 12 ++----------
 1 file changed, 2 insertions(+), 10 deletions(-)

diff --git a/audio_hw.c b/audio_hw.c
index 04846fa..ed98a45 100644
--- a/audio_hw.c
+++ b/audio_hw.c
@@ -179,8 +179,7 @@ static void do_in_standby(struct stream_in *in)
 static int start_output_stream(struct stream_out *out)
 {
     struct audio_device *adev = out->dev;
-    int ret;
-    
+
     ALOGV("start_output_stream");
     if (out->unavailable) {
 	ALOGV("start_output_stream: output not available");
@@ -210,7 +209,6 @@ static int start_output_stream(struct stream_out *out)
 static int start_input_stream(struct stream_in *in)
 {
     struct audio_device *adev = in->dev;
-    int ret;
 
     in->pcm = pcm_open(PCM_CARD, PCM_DEVICE, PCM_IN, in->pcm_config);
     if (!in->pcm) {
@@ -330,10 +328,8 @@ static char *out_get_parameters(const struct audio_stream *stream __unused, cons
     return strdup("");
 }
 
-static uint32_t out_get_latency(const struct audio_stream_out *stream)
+static uint32_t out_get_latency(const struct audio_stream_out *stream __unused)
 {
-    struct stream_out *out = (struct stream_out *)stream;
-    struct audio_device *adev = out->dev;
 
     ALOGV("out_get_latency");
     return (pcm_config_out.period_size * OUT_PERIOD_COUNT * 1000) / pcm_config_out.rate;
@@ -554,7 +550,6 @@ static ssize_t in_read(struct audio_stream_in *stream, void* buffer,
     int ret = 0;
     struct stream_in *in = (struct stream_in *)stream;
     struct audio_device *adev = in->dev;
-    size_t frames_rq = bytes / audio_stream_in_frame_size(stream);
 
     /*
      * acquiring hw device mutex systematically is useful if a low
@@ -773,7 +768,6 @@ static int adev_open_input_stream(struct audio_hw_device *dev,
 
     struct audio_device *adev = (struct audio_device *)dev;
     struct stream_in *in;
-    int ret;
 
     *stream_in = NULL;
 
@@ -810,7 +804,6 @@ static void adev_close_input_stream(struct audio_hw_device *dev __unused,
                                    struct audio_stream_in *stream)
 {
     ALOGV("adev_close_input_stream...");
-    struct stream_in *in = (struct stream_in *)stream;
 
     in_standby(&stream->common);
     free(stream);
@@ -840,7 +833,6 @@ static int adev_open(const hw_module_t* module, const char* name,
     ALOGV("adev_open: %s", name);
 
     struct audio_device *adev;
-    int ret;
     int card = 0;
     char mixer_path[PATH_MAX];
 
-- 
2.7.4

